### PR TITLE
1011 unique no generation

### PR DESCRIPTION
**Pull request by xDAnkit**
- Added new unique id generation method
- Added comments
- Code formatting

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,9 +16,19 @@ const add = (a, b) => {
...
+};
 
-console.log(add(2, 90)) // 92
-console.log(substract(21, 10))
+/**
+ * @description Method to get unique number
+ * @returns
+ */
+const getUniqueNumber = () => {
+  const result = new Date().getTime();
```
Comment (Ashwini1011) [inline index.js]: @xDAnkit don't you think this 'date().getTime' will not return a unique time.

**Review by Ashwini1011 — COMMENTED** [review]
Please review my comments.

File: index.js (hunk 1 above)
Comment (xDAnkit) [reply index.js]: Yes @Ashwini1011 , you're right. Let me update the logic.

Comment (xDAnkit) [reply index.js]: Please review the new code @Ashwini1011

File: index.js [hunk 2]
```diff
@@ -22,10 +22,17 @@ const substract = (a, b) => {
...
-const getUniqueNumber = () => {
-  const result = new Date().getTime();
+const getUniqueNumber = (length = 5) => {
+  let result = "";
+  const characters = "ABCDEFGHJKMNPQRSTUVWXYZabcdefghjkmnpqrstuvwxyz0123456789";
+  const charactersLength = characters.length;
+  for (var i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  result = new Date().getTime() + "-" + result;
```
Comment (Ashwini1011) [inline index.js]: @xDAnkit can we use template string over here?

**Review by Ashwini1011 — COMMENTED** [review]
Please check my comments.

File: index.js (hunk 2 above)
Comment (xDAnkit) [reply index.js]: Yes, we can. Doing it right away

**Review by Ashwini1011 — APPROVED** [review]
Looks good to me.

**Pull request merged**